RELEASE_NOTES

```diff
@@ -1,10 +1,15 @@
 ========================================================================
-Amazon CloudWatch Agent 1.300046.0 (2024-09-04)
+Amazon CloudWatch Agent 1.300046.0 (2024-09-05)
 ========================================================================
-Enhancements:
+Features:
 * Attach Entity field to CloudWatch Logs PutLogEvent API
+
+Enhancements:
 * Increase retry delay on PLE for certain exceptions
 
+Bug Fixes:
+* Fix duplicate log events
```

Comment:
do we need to include this? This bug was never deployed publicly, it was caught in our testing

Reply:
Fair point, fixed.